prdoc/pr_6540.prdoc

```diff
@@ -0,0 +1,16 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Only allow apply slash to be executed if the slash amount is atleast ED
+
+doc:
+  - audience: Runtime User
+    description: |
+      This change prevents `pools::apply_slash` from being executed when the pending slash amount of the member is lower
+      than the ED. With this change, such small slashes will still be applied but only when member funds are withdrawn.
+
+crates:
+- name: pallet-nomination-pools-runtime-api
+  bump: patch
+- name: pallet-nomination-pools
+  bump: major
```

Comment:
@seadanda I assume it will only patch the existing release.

Also, practically this may not be a breaking change (I think). The client gets the new type information from the metadata, and the module error is only really for error description. Changes in Storage or RuntimeCall are more critical as some dapp's business logic may depend on it. 

I could make one of the following change if that helps:
- Add the new error variant to the end so existing indexes are not affected.
- The backport version does not add the new error and reuses one of the existing errors.